build/scripts/build-ccutil.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2023-2024 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Utility script build html previews with referenced images
+# Requires: asciidoctor - see https://docs.asciidoctor.org/asciidoctor/latest/install/linux-packaging/
```

Comment:
Please clean up the copyright date and require here, which is no longer correct. 

Can you also include in this PR the changes needed to run the container within a GH action instead of the current action that uses cmndline asciidoctor?

Reply:
I was thinking: deliver soon, deliver often. 

1. Deliver quickly this first PR to have something that works on people's laptop. 
2. Then enhance to have something that works in GitHub. This will require changes in the script.
 
WDYT @nickboldt?

Reply:
sure, as long as the JIRA for this task isn't closed until you're done implementing this as a CI / PR check. Expecting humans to run this manually will lead to humans forgetting to run this manually.